cypress/integration/code.spec.js

```diff
@@ -0,0 +1,42 @@
+/// <reference types="Cypress" />
+/* global cy, context, it */
+
+context('Open Source Code', () => {
+  it('Loads the English page', () => {
+    cy.visit('http://localhost:4000/ore-ero/en/open-source-codes.html');
+    cy.get('#wb-cont').contains('Open Source Code');
+  });
+
+  it('Dynamically filters on the English page', () => {
+    cy.visit('http://localhost:4000/ore-ero/en/open-source-codes.html');
+    cy.get('#dataset-filter_filter')
+      .find('input')
+      .type('youcanbenefit');
+
+    cy.get('#dataset-filter')
+      .find('tbody>tr')
+      .first()
+      .find('td>a')
+      .first()
+      .contains('YouCanBenefit');
+  });
+
+  it('Loads the French page', () => {
+    cy.visit('http://localhost:4000/ore-ero/fr/codes-source-ouverts.html');
+    cy.get('#wb-cont').contains('Code source ouvert');
```

Comment:
This is the failing test, could it be because it is case sensitive? The actual value to `h1#wb-cont` on that page is "Code Source Ouvert" with each word capitalized.